interface.go

```diff
@@ -216,7 +216,7 @@ type LoopInQuoteRequest struct {
 	// include the swap and miner fee.
 	Amount btcutil.Amount
 
-	// HtlcConfTarget specifies the targeted confirmation target for the
+	// HtlcConfirmations specifies the targeted confirmation target for the
```

Comment:
same

Reply:
ah, this is actually find and replace gone wild, shouldn't have been renamed in the first place, removed.